charts/bulk-scan-processor/values.yaml

```diff
@@ -44,6 +44,10 @@ java:
     "bulk-scan":
       secrets:
         - processor-POSTGRES-PASS
+        - processor-POSTGRES-USER
+        - processor-POSTGRES-DATABASE
+        - processor-POSTGRES-HOST
+        - processor-POSTGRES-PORT
```

Comment:
why do we need to add this for staging ?

Reply:
@nitinprabhu Oh, yes